Admin/Get-EASMailboxLogs.ps1

```diff
@@ -2,13 +2,34 @@
 # Licensed under the MIT License.
 
 # Powershell script to enable and collect EAS mailbox logs.
+[CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'Medium')]
 param(
     [Parameter(Mandatory = $true)] [string[]] $Mailbox = "",
     [ValidateScript({ Test-Path $_ })]
     [Parameter(Mandatory = $true)] [string] $OutputPath = (Get-Location).Path,
-    [Parameter(Mandatory = $false)] [int] $Interval = 30
+    [Parameter(Mandatory = $false)] [int] $Interval = 30,
+    [Parameter(Mandatory = $false)] [nullable[bool][]] $EnableMailboxLoggingVerboseMode = $null
 )
 
+#parse $EnableMailboxLoggingVerboseMode and convert it boolean value(s)
+$NeedChangeConfig = $false
+switch ($EnableMailboxLoggingVerboseMode) {
+    $true { $NeedChangeConfig = $true; $EnableVerboseLogging = "true" }
+    $false { $NeedChangeConfig = $true; $EnableVerboseLogging = "false" }
+}
+
+#Override EnableMailboxLoggingVerboseMode key's value with EnableVerboseLogging
+if ($NeedChangeConfig) {
+    if ($PSCmdlet.ShouldProcess("Set EnableMailboxLoggingVerboseMode attribute to $EnableVerboseLogging in $env:ExchangeInstallPath" + "ClientAccess\Sync\web.config", 'TARGET', 'OPERATION')) {
+        [xml]$web = Get-Content $env:ExchangeInstallPath"ClientAccess\Sync\web.config"
+        Copy-Item $env:ExchangeInstallPath"ClientAccess\Sync\web.config" -Destination $env:ExchangeInstallPath"ClientAccess\Sync\web.config.bak"
+        $web.SelectSingleNode('//add[@key="EnableMailboxLoggingVerboseMode"]').Value = $EnableVerboseLogging
+        $web.Save($env:ExchangeInstallPath + "ClientAccess\Sync\web.config")
+    } else {
+        break
```

Comment:
why is the `break` here? I think this needs to be removed.

Reply:
To exit script without enabling logging.

Reply:
Then we should use `exit` instead of break.